interface/modules/custom_modules/oe-module-claimrev-connect/src/Eligibility.php

```diff
@@ -0,0 +1,237 @@
+<?php 
+use OpenEMR\Modules\ClaimRevConnector\EligibilityData;
+use OpenEMR\Modules\ClaimRevConnector\EligibilityInquiryRequest;
+use OpenEMR\Modules\ClaimRevConnector\SubscriberPatientEligibilityRequest;
+use OpenEMR\Modules\ClaimRevConnector\EligibilityObjectCreator;
+use OpenEMR\Modules\ClaimRevConnector\ValueMapping;
+
+
+$insurance = EligibilityData::getInsuranceData($pid);
+?>
+
```

Comment:
for this script:
Translate labels
html escpaping
use require instead of include

Reply:
done